proving_system/stark/src/air/frame.rs

```diff
@@ -7,15 +7,29 @@ use super::trace::TraceTable;
 
 #[derive(Clone, Debug)]
 pub struct Frame<F: IsField> {
-    data: Vec<FieldElement<F>>,
+    // Vector of rows
+    data: Vec<Vec<FieldElement<F>>>,
```

Comment:
Why not keep the current `data` field? It can already handle multiple columns, since the number of columns is fixed in every `AIR`. The `get_row` methods already take that into account using the `row_width` field.